extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -81,6 +86,10 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
     /** @private {?Element} */
     this.ampAnalyticsElement_ = null;
+
+    // Will initiate the refresh lifecycle iff the slot has been enabled to do
+    // so through an appropriate data attribute, or a page-level meta tag.
+    new RefreshManager(this).initiateRefreshCycle();
```

Comment:
Should this wait until after layout otherwise its possible refresh interval would include time to fetch and render first ad slot?